spec/features/flowdock_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature "flowdock api" do
+  context "authorized user" do
+
+    scenario 'can successfully post questions on flowdock', :vcr do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.